articles/documentation-style-guide.md

```diff
@@ -0,0 +1,331 @@
+
+### Tutorials section
+
+For the tutorials section, we suggest creating individual pages for various applications where the package is likely to be used. For each application, guide learners through the workflow step by step, organizing it into separate exercises with clear objectives and visible results. This structure makes the content more accessible and manageable for learners, especially for complex workflows.
+
+#### Submodule
+For packages with extensive tutorials, multimedia and other large files, a separate repository can be created and imported as a submodule.
+
+A tutorial submodule can be added with the following command in the `docs` directory:
+
+```powershell
+git submodule add https://github.com/bonsai-rx/machinelearning-examples
+```
+
+In addition, the `docfx.json` file needs to be modified to import the relevant resources. For an example of how to setup a Tutorial submodule, refer to the [Bonsai Machine Learning Package](https://github.com/bonsai-rx/machinelearning) package and its submodule [Machine Learning Examples](https://github.com/bonsai-rx/machinelearning-examples).
+
+## Article formatting guide 
+
+In docfx, articles are written in [Markdown](https://dotnet.github.io/docfx/docs/markdown.html?tabs=linux%2Cdotnet) and rendered with the [Markdig](https://github.com/xoofx/markdig) parsing engine. Refer to these links for more information on how to format content in markdown.
+
+When writing articles, please follow the [MSDN writing tips](https://learn.microsoft.com/en-us/style-guide/global-communications/writing-tips). In particular:
+
+- Keep article and section titles short and succinct so that the table of contents that appears on the left and right sidebar are easier to read (and also to assist in machine translation)
+- Reuse operator names, properties, and descriptions in the articles and titles (do not use synonyms) so that readers may more easily follow and refer to them.
+- Use the imperative style i.e. "Link hardware triggers" rather than "Linking hardware triggers".
+
+### Linking content 
+When linking to content (either images, or other articles, or workflow containers) use relative paths in this format:
+
+**Example:**
+
+For content in the same folder:
+
+```markdown
+./content.md
+```
+
+For content in a different folder, use `../` to navigate up to the parent folder, and then specify the folder with the content like this:
+
+```markdown
+../images/bonsai-screenshot.svg
+```
+
+### Cross-references and highlights
+
+When working on an article, first check [the main documentation](https://bonsai-rx.org/docs/) to see what written material might already exist that could be linked if it would be helpful and to avoid duplication. Link resources like websites for external libraries for users to explore for more information.
```

Comment:
```suggestion
When working on an article, first check [the main documentation](https://bonsai-rx.org/docs/) to see what written material might already exist that could be linked if it would be helpful and to avoid duplication. Link resources like websites for external libraries which may be useful to explore for more information.
```